pom.xml

```diff
@@ -208,7 +208,7 @@
                     <dependency>
                         <groupId>com.puppycrawl.tools</groupId>
                         <artifactId>checkstyle</artifactId>
-                        <version>8.26</version>
+                        <version>8.23</version>
```

Comment:
Note: I'm rolling this back to 8.23 because the xml config errors with 8.26

Reply:
Yeah, I just had to do that this morning on another project. :(